internal/pkg/service/stream/definition/repository/branch/branch_create.go

```diff
@@ -18,29 +18,29 @@ import (
 func (r *Repository) Create(input *definition.Branch, now time.Time, by definition.By) *op.AtomicOp[definition.Branch] {
 	k := input.BranchKey
 	var created definition.Branch
-	var deleted *op.KeyValueT[definition.Branch]
+	var deleted *definition.Branch
```

Comment:
Now, we can use `GetOrNil`, instead of more complicated `GetKV` (we don't need KV metadata).